docs/source/supported_features.rst

```diff
@@ -0,0 +1,97 @@
+Supported Features
```

Comment:
thoughts on renaming this "Supported Properties" or something of that nature? could be slightly confusing to use feature as both the overall feature set and as a subset (autograd/torchscript)

Reply:
Yes! I was also feeling that "feature" is overloaded here. Thanks for the suggestion. Will do.

Reply:
Updated. It still uses the ambiguous `feature` term. Let me know what you think.

Reply:
I am thinking to mention numerical compatibility with other libraries such as librosa and Kaldi, so let me follow-up on this one.